packages/mock-block-dock/dev/dev-app.tsx

```diff
@@ -1,42 +1,61 @@
-import { Entity } from "@blockprotocol/graph";
+import { extractBaseUri } from "@blockprotocol/type-system/slim";
 import { ChangeEvent, FunctionComponent, useState } from "react";
 import { createRoot } from "react-dom/client";
 
 import { MockBlockDock } from "../src";
+import { entityTypes } from "../src/data/entity-types";
+import { propertyTypes } from "../src/data/property-types";
 import { TestCustomElementBlock } from "./test-custom-element-block";
 import testBlockString from "./test-html-block/block.html?raw";
 import { TestReactBlock } from "./test-react-block";
 
 const node = document.getElementById("app");
 
-type TestBlockType =
-  | "react"
-  | "custom-element"
-  | "html-at-url"
-  | "html-as-string";
-
 const blockEntityMap = {
   react: {
-    entityId: "entity-react",
-    entityTypeId: "test-type-1",
-    properties: { name: "World" },
+    metadata: {
+      editionId: {
+        baseId: "entity-react",
+        versionId: new Date().toISOString(),
+      },
+      entityTypeId: entityTypes.testType.$id,
+    },
+    properties: { [extractBaseUri(propertyTypes.name.$id)]: "World" },
   },
   "custom-element": {
-    entityId: "entity-custom-element",
-    entityTypeId: "test-type-1",
-    properties: { name: "World" },
+    metadata: {
+      editionId: {
+        baseId: "entity-custom-element",
+        versionId: new Date().toISOString(),
+      },
+      entityTypeId: entityTypes.testType.$id,
+    },
+    properties: { [extractBaseUri(propertyTypes.name.$id)]: "World" },
   },
   "html-at-url": {
-    entityId: "entity-html-as-url",
-    entityTypeId: "test-type-1",
-    properties: { name: "World" },
+    metadata: {
+      editionId: {
+        baseId: "entity-html-as-url",
+        versionId: new Date().toISOString(),
+      },
+      entityTypeId: entityTypes.testType.$id,
+    },
+    properties: { [extractBaseUri(propertyTypes.name.$id)]: "World" },
   },
   "html-as-string": {
-    entityId: "entity-html-as-string",
-    entityTypeId: "test-type-1",
-    properties: { name: "World" },
+    metadata: {
+      editionId: {
+        baseId: "entity-html-as-string",
+        versionId: new Date().toISOString(),
+      },
+      entityTypeId: entityTypes.testType.$id,
+    },
+    properties: { [extractBaseUri(propertyTypes.name.$id)]: "World" },
   },
-} as Record<string, Entity>;
+} as const;
+// } as const satisfies Record<string, Entity>;
```

Comment:
It would be nice to start using these appropriately, it's not really superfluous as it actually removes the need for casts and assertions in certain usages.

I can remove it if you'd prefer to not have commented out code (I tend to err on the side of removing it too), or I can put a todo about reintroducing this when satisfies is better supported?

Reply:
I'm not sure why `satisfies` wouldn't work here. Leave it in and I'll look at it when I'm doing stuff in MBD